### ISSUE TITLE
NFSCUnlimited only supports 1.4 English NoCD exe?

### ISSUE THREAD

**Issue by KyoriAsh** [nlgzrgn/NFSCUnlimiter#1]
![image](https://github.com/user-attachments/assets/8fa5a0fa-b80c-41f6-b20c-de4e18753bc3)

Is it possible to support 1.3 NoCD Asian exe? This is due to 1.4 English NoCD exe is not able to support Asian languages due to missing pointers pointing to Asian language files such as in the Language folder.

**Comment by Sh2dow** [comment]
Isn't it easier to make a patch for 1.4 to support Asian languages instead of asking every single mod maker for support 1.3?

**Comment by bluesky-dev12** [comment]
For that you would to change offsets from memory of stuff for carbon* 1.3 asian to be supported...

**Comment by Sh2dow** [comment]
> mw 1.3
isn't it all about nfsc?

**Comment by bluesky-dev12** [comment]
Yeah sorry but still. Every recompilation of code changes the adress of stuff. Theres why it's locked to specify version.
